spec/form_models/publishers/job_listing/copy_vacancy_form_spec.rb

```diff
@@ -1,26 +1,25 @@
 require "rails_helper"
 
-RSpec.describe CopyVacancyForm, type: :model do
+RSpec.describe Publishers::JobListing::CopyVacancyForm, type: :model do
+  subject { described_class.new({ job_title: job_title }) }
+
   describe "validations" do
     describe "#job_title" do
-      let(:copy_form) { CopyVacancyForm.new(job_title: job_title) }
-
       context "when title is blank" do
         let(:job_title) { nil }
 
         it "requests an entry in the field" do
-          expect(copy_form.valid?).to be false
-          expect(copy_form.errors.messages[:job_title][0])
-            .to eq("Enter a job title")
+          expect(subject.valid?).to be false
```

Comment:
This, and in many other places, we could have a shorter, more readable spec with:

it { is_expected.to validate_presence_of(:job_title).with_message("Enter a job title") }

Reply:
I think that makes sense, let's agree an approach and make a ticket for it 👍